errors/mappers/pqerrors/pqerrors.go

```diff
@@ -0,0 +1,117 @@
+package pqerrors
+
+import (
+	"context"
+
+	"github.com/lib/pq"
+	"google.golang.org/grpc/codes"
+
+	"github.com/infobloxopen/atlas-app-toolkit/errors"
+)
+
+const (
+	msgForeignKeyViolation = "Cannot insert The '%s' object as it does not refer to a valid '%s' object."
```

Comment:
Looks like `The` is redundant here.
How about "Cannot insert object '%s' as it does not refer to a valid object '%s'."

Reply:
same, '%s' stands for object type name.